docs/getting-started/quickstart.md

```diff
@@ -7,58 +7,57 @@ Having [NodeJS](https://nodejs.org/) installed, just type :
 ```bash
 npm install dash
 ```
+
 ## Initialization
 
 Let's create a Dash SDK client instance specifying both our mnemonic and the schema we wish to work with.
 
 ```js
-const Dash = require("../src");
+const Dash = require('dash');
 const opts = {
-  network: 'testnet',
   apps: {
     dashpay: {
-      contractId:1234,
-      schema: require('schema.json')
+      contractId: '77w8Xqn25HwJhjodrHW133aXhjuTsTv9ozQaYpSHACE3',
     },
   },
   wallet: {
     mnemonic: "arena light cheap control apple buffalo indicate rare motor valid accident isolate",
   },
 };
 const client = new Dash.Client(opts);
-client.isReady().then(()=>{
-    const {account} = client;
-    // Do something
- });
+client.getWalletAccount().then(async (account) => {
+  // Do something
+})
 ```
 
 Quick note :
-- If no mnemonic is provided, the subinstance `client.Wallet` will not be initiated (write function for platforms won't be usable).
 
-If you do not have any mnemonic, you can pass `null` to get one generated or omit that parameter to only use Dash.Client in `read-only`.  
+- If no mnemonic is provided, the sub-instance `client.Wallet` will not be initialized (writing  capabilities of Dash Platform won't be usable).
 
+If you do not have a mnemonic, you can pass `null` to have one generated or omit that parameter to only use Dash.Client for `read-only` operations.
 
 ## Make a payment
 
 ```js
-client.isReady().then(()=>{
-     const {account} = client;
-
-    account
-      .createTransaction({
-        recipient:{address:'yLptqWxjgTxtwKJuLHoGY222NnoeqYuN8h', amount:0.12}
-      })
-      .then(account.broadcastTransaction);
+client.getWalletAccount().then(async (account) => {
+  const transaction = account.createTransaction({
+    recipient: 'yixnmigzC236WmTXp9SBZ42csyp9By6Hw8',
+    amount: 0.12,
   });
+  account.broadcastTransaction(transaction);
+});
 ```
 
-## Read a document 
+## Read a document
 
-At time of writing, you will need to have registered dashpay yourself, see on [publishing a new contract](/examples/publishing-a-new-contract.md).
+At the time of writing, you will need to have registered a data contract yourself. See [publishing a new contract](/examples/publishing-a-new-contract.md).
 
 ```js
-client.isReady().then(async ()=>{
-    const {account} = client;
-    const bobProfile = await account.platform.documents.fetch('dashpay.profile', {name:'bob'})
-  });
+
+client.platform.documents.get(
+  'tutorialContract.note',
+  { limit: 1 }, // Only retrieve 1 document
+).then(async (documents) => {
```

Comment:
This can actually just be simplified into `then(console.log)`

Reply:
Good to know :+1: